packages/components/src/index.ts

```diff
@@ -2,13 +2,17 @@ export * from "./badge/index.ts";
 export * from "./buttons/index.ts";
 export * from "./checkbox/index.ts";
 export * from "./chip/index.ts";
+export * from "./divider/index.ts";
 export * from "./errorMessage/index.ts";
+export * from "./header/index.ts";
```

Comment:
Components have their parent folder starting with an uppercase letter, as for the Divider.